sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1639,6 +1657,13 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val isParquetInt96WriteEnabled: Boolean = get(ENABLE_PARQUET_INT96_WRITE)
 
+  lazy val parquetReaderFooterType: ParquetFooterReaderType.Value = {
+    get(PARQUET_READER_FOOTER_TYPE) match {
+      case "NATIVE" => ParquetFooterReaderType.NATIVE
+      case _ => ParquetFooterReaderType.JAVA
```

Comment:
I think this should fail, rather than assume anything not "NATIVE" means Java.  For example, if the user specified "NATIV" it will silently assume they meant "JAVA".

Reply:
I did it this way because the config should be enforcing some of this, but I will be more defensive because we know that the config is not 100% perfect in how it checks for things.